client/coral-sign-in/containers/SignInContainer.js

```diff
@@ -119,8 +119,7 @@ class SignInContainer extends Component {
     }
   }
 
-  handleSignIn(e) {
-    e.preventDefault();
```

Comment:
Careful removing an `e.preventDefault()` because the event bubles up and it might refreshes the page or scrolls to top in Chrome or Safari